styleguide/_layouts/default.jade

```diff
@@ -46,8 +46,25 @@ html(lang="en")
 
     block content
 
-    a(href="https://github.com/rackerlabs/zoolander")
-      img(style="position: absolute; top: 40px; right: 0; border: 0;", src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png")
 
     script(src="https://code.jquery.com/jquery-1.12.3.min.js" integrity="sha256-aaODHAgvwQW1bFOGXMeX+pC4PZIPsvn2h1sArYOhgXQ=" crossorigin="anonymous")
     script(src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous")
+    script(src="http://code.jquery.com/jquery.min.js")
+
+    script.  
```

Comment:
Instead of this inline JS, @bighappyface recommends we strip this out and just create a generic "zoolander.js" file and include it here with any customizations.